content-resources/src/main/resources/flyway/oskari/V1_44_0__update_backendstatus_schema.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+
+-- drop views, handle logic in code not db
+DROP VIEW IF EXISTS portti_backendstatus_allknown;
+DROP VIEW IF EXISTS portti_backendalert;
+
+-- modernise the name of the table
+ALTER TABLE portti_backendstatus RENAME TO oskari_backendstatus;
+
+-- drop columns that are not needed
+ALTER TABLE oskari_backendstatus DROP COLUMN id;
+ALTER TABLE oskari_backendstatus DROP COLUMN source;
```

Comment:
Source can be used for identifying which system triggered the status. I would keep it.